### PR TITLE
'NGFW Scan' XSIAM Test Playbook

### PR DESCRIPTION
**Pull request by TalNos**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-hq.paloaltonetworks.local/browse/CIAC-7783

## Description
Test Playbook for 'NGFW Scan' XSIAM Playbook

## Screenshot:
Test_Playbook_-_NGFW_Scan_Wed_Sep_27_2023

### CODE REVIEW

File: Packs/Core/.secrets-ignore [hunk 1]
```diff
@@ -83,4 +83,5 @@ SailPoint
 dummy@dummy.com
 dummy1@dummy.com
 dummy2@dummy.com
-dummy3@dummy.com
\ No newline at end of file
+dummy3@dummy.com
+80.66.75.36
```
Comment (altmannyarden) [inline Packs/Core/.secrets-ignore:87]: why you added this IP?

File: Tests/conf.json [hunk 2]
```diff
@@ -20,6 +20,18 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+        {
+            "playbookID": "Test Playbook - NGFW Scan",
+			"integrations": [
+				"VirusTotal (API v3)",
+				"Cortex XDR - IR"
```
Comment (altmannyarden) [inline Tests/conf.json:27]: why you need cortex XDR IR integration for XSIAM playbook?
same for the instance name

File: Packs/Core/.secrets-ignore (hunk 1 above)
Comment (TalNos) [reply Packs/Core/.secrets-ignore:87]: cause the PR validation failed on 'secret detection' so I added it to the ignore list.

File: Tests/conf.json (hunk 2 above)
Comment (TalNos) [reply Tests/conf.json]: removed

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**